여경/week2_array.js

```diff
@@ -0,0 +1,9 @@
+function solution(array, n){
+    let answer = 0;
+   array.forEach(function(item){
+     if(item === n){
+       answer++;
```

Comment:
자바스크립트에서는 사이드이펙트가 있을 수 있어 ++ 연산자보다는 += 1 연산자를 권장하는편이에요!
```suggestion
       answer += 1;
```
이번기회에 사이드 이펙트에 대해서 찾아보는건 좋은 학습의 기회가 될수도..